attestation/src/measurement.rs

```diff
@@ -149,9 +172,33 @@ impl<'a, D: Digest> AttestationManager<D> {
             );
         }
 
-        AttestationManager {
+        // Derive a TVM specific CDI from the TSM CDI.
+        let kdf = Hkdf::<D, H>::new(Some(&vm_id.to_le_bytes()), cdi);
```

Comment:
so the recommendation for HKDF salt is to be a fixed random value or NULL - the vm_id could be used as part of the info

Reply:
Fixed as well.